# Fall 2022 Data Science Intern Challenge 

On Shopify, we have exactly 100 sneaker shops, and each of these shops sells only one model of shoe. We want to do some analysis of the average order value (AOV). When we look at orders data over a 30 day window, we naively calculate an AOV of \\$3145.13. Given that we know these shops are selling sneakers, a relatively affordable item, something seems wrong with our analysis. 


In [32]:
import pandas as pd
import numpy as np

In [30]:
df = pd.read_csv("Shopify_data.csv")

In [31]:
df["order_amount"].describe()

count      5000.000000
mean       3145.128000
std       41282.539349
min          90.000000
25%         163.000000
50%         284.000000
75%         390.000000
max      704000.000000
Name: order_amount, dtype: float64

As we see here, our average order value is indeed \\$3145.13 which does not make sense as we are looking at shoes. Furthermore, our standard deviation is outrageously large, at \\$41282 . This suggests the existence of outliers in the data (and potentially fraud). However when looking at quantiles, we see that the median order value (MOV) is much more reasonable at \\$284. Therefore, we would want to report median order value at \\$284